### PR TITLE
Add NGINX

### PR DESCRIPTION
**Pull request by matkrzeminski**
This PR adds NGINX.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -135,6 +133,18 @@ services:
     env_file:
         - .env
 
+  nginx:
+    image: bitnami/nginx
+    volumes:
+      - ./bilara_server_block.conf:/opt/bitnami/nginx/conf/server_blocks/bilara_server_block.conf:ro
+      - ./frontend:/usr/share/nginx/html:ro
```
Comment (mkorlaga) [inline docker-compose.yml:140]: Weird landing location why not simplier?

**Comment by mkorlaga** [comment]
I would change the banking of config file so it is more obvious

**Comment by mkorlaga** [comment]
I know that we don't have logger jest but this also works be useful to have specified

**Comment by matkrzeminski** [comment]
I followed Bitnami's naming convention when it comes to the config file.

**Review by mkorlaga — APPROVED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -135,6 +133,19 @@ services:
...
+  nginx:
+    image: bitnami/nginx
+    volumes:
+      - ./bilara_server_block.conf:/opt/bitnami/nginx/conf/server_blocks/bilara_server_block.conf:ro
+      - ./nginx_logs:/var/log/nginx
+      - ./frontend:/var/www/html:ro
+    ports:
+      - ${DOCKER_BACKEND_PORT}:${NGINX_PORT}
+    depends_on:
+      - bilara-backend
```
Comment (mkorlaga) [inline docker-compose.yml:145]: During Bilara reboot, nothing is served from nginx until bilara-backed will start (which itself depend on elastic and data population)

**Pull request merged**